modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: PROCEDURE
+[id="cluster-logging-collector-log-forward-gcp_{context}"]
+= Forwarding logs to Google Cloud Platform
+
+You can forward logs to link:https://cloud.google.com/logging/docs/basic-concepts[Google Cloud Logging] in addition to, or instead of, the internal default {product-title} log store.
+
+[NOTE]
+====
+Using this feature with fluentd is not supported.
+====
+
+.Prerequisites
+* {logging-title-uc} Operator 5.5.1 and later
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  outputs:
+    - name: gcp-1
+      type: googleCloudLogging
+      secret:
+        name: gcp-secret
+      googleCloudLogging: <1>
+        projectId : "openshift-gce-devel"
+        logId : "app-gcp" <2>
+  pipelines:
+    - name: test-app
+      inputRefs:
+        - application
+          #- infrastructure
+      outputRefs:
+        - gcp-1
+----
+<1> Exactly one of projectId, folderId, organizationId, or billingAccountId must be set depending on where you want to store your logs in the link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy[GCP resource hierarchy].
```

Comment:
```suggestion
<1> Set either a `projectId`, `folderId`, `organizationId`, or `billingAccountId` stanza, depending on where you want to store your logs in the link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy[GCP resource hierarchy].
```
- Used active voice, put backticks around items that appear in code, and made them adjectives of the noun, "stanza."